tests/views/test_api.py

```diff
@@ -1,29 +1,326 @@
-from mock import patch
+from datetime import datetime
 
-from gamecenter.api import views
 from gamecenter.api.models import Score
 from gamecenter.core.utils_test import BaseTestCase
 from gamecenter.core.models import DB
 
 
+URL_PREFIX = "https://tmwild.com/api/"
+
+
 class APIViewsTest(BaseTestCase):
+    starting_data = [  # don't modify this list
+        Score(creation_date=datetime(2015, 4, 19), user_id=1, score=21),
```

Comment:
the `creation_date` field should be `created_at` to align with the column you added in api/models.py